src/uct/base/uct_iface.h

```diff
@@ -428,7 +429,12 @@ typedef struct uct_iface_mpool_config {
     {_prefix "BUFS_GROW", UCS_PP_QUOTE(_dfl_grow), \
      "How much buffers are added every time the " _mp_name " memory pool grows.\n" \
      "0 means the value is chosen by the transport.", \
-     (_offset) + ucs_offsetof(uct_iface_mpool_config_t, bufs_grow), UCS_CONFIG_TYPE_UINT}
+     (_offset) + ucs_offsetof(uct_iface_mpool_config_t, bufs_grow), UCS_CONFIG_TYPE_UINT}, \
```

Comment:
the documentation and naming of this parameter is not correct anymore: it's not how much buffers are added every time; it's the initial chunk size, and the number of added buffers grows by grow_factor every time

Reply:
fixed.